bitbots_docs/docs/manual/tutorials/vscode-ros2.rst

```diff
@@ -70,8 +70,6 @@ An example vscode launch configuration and the corresponding python lauch file c
                         "base_link_frame": "base_link",
                         "odom_frame": "odom",
                         "map_frame": "map",
-                        "ball_frame": "ball",
-                        "ball_approach_frame": "ball_approach_frame",
                         "base_footprint_frame": "base_footprint"
                     }],
                 remappings=[("/head_motor_goals", "/DynamixelController/command")]
```

Comment:
Please update this entire example, as we don't have the head-behavior anymore.

Reply:
Ahhh

Reply:
I didn't reach the tools repo

Reply:
The entire file should be updated with our new configuration… Let's do that after the cleanup. I have created this issue: https://github.com/bit-bots/bitbots_tools/issues/160